application/main.py

```diff
@@ -91,6 +105,7 @@ def send_request(self, request: bytes):
     def build_request(self):
         origin = self.fetch_coords(self.request["origin"])
         destinations = self.fetch_destinations(self.request["destinations"])
+        print(destinations)
```

Comment:
Do we really want this in any kind of output when there could be 1000's?

Reply:
agree, removing